pkg/monitor/monitor/util.go

```diff
@@ -1149,14 +1165,18 @@ func defaultTidbMonitor(monitor *v1alpha1.TidbMonitor) {
 	if monitor.Spec.PVReclaimPolicy == nil {
 		monitor.Spec.PVReclaimPolicy = &retainPVP
 	}
+	// If tidb clusters is null,we set a zero slice.
```

Comment:
why?

Reply:
Because if `spec.clusters` field is null ,update tidb status will throw nil exception. So we need to set an empty array.

Reply:
sorry, I can not catch the point, can you clarify it?

Reply:
If we don't set empty array, operator update will throw below exception.
![image](https://user-images.githubusercontent.com/13257127/139857593-35e2a0a2-c198-46af-b15d-8e986213b4f9.png)

Reply:
add `omitempty` tag in `TidbMonitorSpec` for `Clusters` and drop this code?

Reply:
ok